specification/azurestackhci/resource-manager/readme.md

```diff
@@ -64,14 +64,30 @@ suppressions:
     reason: The linter is mistakenly thinking that paths for a singular resource that is always named default, like "/{resourceUri}/providers/Microsoft.AzureStackHCI/virtualMachineInstances/default," is for a collection of resources.
     from: virtualMachineInstances.json
   - code: TopLevelResourcesListBySubscription
-    reason: There is a 1:1 relationship between HybridCompute Machines and AzureStackHCI VirtualMachineInstances
+    reason: It is reporting issue for proxy extension resource which doesn't have use case to ListBySubscription as this resource will always tied to one parent resource only. Additionally, there is a 1:1 relationship between HybridCompute Machines and AzureStackHCI VirtualMachineInstances.
```

Comment:
Again I hope this suppression shouldn't be needed.

I wonder if the leading slash before '{resourceUri}': i.e. '/{resourceUri}' i the swagger paths, might be the actual cause of some of these linter errors.

Reply:
(I can understand in this case we probably don't need list by subscription though.)

Reply:
The linter actually indicates that the property is not allowed without the leading slash. We will keep this suppression for now, as the cluster resources require this suppression as well. 

Let me know if you would like to revisit it.

#won'tfix

Reply:
Really? I thought we had a linter rule saying the reverse. I'm officially confused! /cc @rkmanda